hw3/01-characters.js

```diff
@@ -1,4 +1,55 @@
 // url for the Thrones API
-const url = 'https://thronesapi.com/api/v2/Characters';
+const url = "https://thronesapi.com/api/v2/Characters";
 
-fetch(url);
+//reference --https://www.javascripttutorial.net/javascript-fetch-api/
+
+let section = document.querySelector("section");
+
+function show(character) {
+  let div = document.createElement("div");
```

Comment:
Consider using a more descriptive name than just "div", this could become confusing when working with more divs.